### DISCUSSION TITLE
Components failing to load in v3.0.9

### DISCUSSION THREAD

**Discussion by CyberPunkCodes** [livewire/livewire#7093] [7. Bugs (archived)] [answered]
### Livewire version

3.0.9

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Follow the instructions to [install Jetstream](https://jetstream.laravel.com/installation.html)

choose `php artisan jetstream:install livewire`

After finishing the remaining steps (install, run dev, migrate), create a new account, and when you hit the dashboard the `navigation-menu` fails to load. In fact, all components fail to load. In this case, removing `@livewire('navigation-menu')` lets the page load, albeit, without the menu lol.

Someone replied to my closed issue over there and said ALL of his components started failing to load on update to 3.0.9 and reverting to 3.0.8 worked.

### Additional information

It seems Livewire is unusable as of v3.0.9 - Reverting back to v3.0.8 works. All components fail to load in 3.0.9.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Omranic** [comment, marked as answer]
Thanks @CyberPunkCodes 

I've successfully replicated this issue and detailed my findings in https://github.com/livewire/livewire/pull/7076#issuecomment-1768027945

Additionally, I've submitted a solution to the Jetstream team. If they validate these insights, my proposed changes should resolve the problem. Here's the link to the pull request: https://github.com/laravel/jetstream/pull/1390.

**Comment by CyberPunkCodes** [comment]
Thanks @Omranic for promptly taking care of this.

It looks like 3.0.10 is a revert and the PR on Jetstream was accepted. I assume you are waiting on it being in the next release, in which you will re-apply the changes for 3.0.11?

How did 3.0.9 pass the tests? I am terrible at writing tests.. but what could be done test wise to catch this in the future? Do the browser tests test on a basic Laravel app or on Livewire? Is that possible?

**Reply by Omranic** [reply]
Hey @CyberPunkCodes, thanks for the inputs.

- Yes, once Laravel Jetstream is tagged, and the commit is available in the next release, I'll re-submit this update again to Livewire.

- Regarding tests, well, this is not an issue with Livewire itself, it was an issue with Jetstream, which is why Livewire tests pass normally, and I wasn't using Jetstream, so unless you're using both Livewire & Jetstream together, you wouldn't be catching this issue.

Hope this helps 🙌
